sandbox/settings.py

```diff
@@ -232,6 +255,44 @@ class Base(Configuration):
     MACHINA_PROFILE_AVATARS_ENABLED = False
     MACHINA_USER_DISPLAY_NAME_METHOD = "get_public_username"
 
+    # Sentry
+    SENTRY_DSN = values.Value(None, environ_name="SENTRY_DSN")
+
+    # pylint: disable=invalid-name
+    @property
+    def ENVIRONMENT(self):
```

Comment:
As you are not reusing this property, you can directly  you its value in the `post_setup` function